Jenkinsfile

```diff
@@ -98,6 +115,26 @@ pipeline {
         } }
       }
     }
+    stage('Smoke Test') {
```

Comment:
I'm a little confused as to why we are calling this stage "Smoke test"
Are we planning to crate a draft snapshot release on GH? 
Is the intention to do some pass-fail automated test with the vsix file in this stage?

Reply:
Yes I believe the goal here was, to quote @crawr:
> Suggestion: Add a new step to publish a vsix file to a libs-snapshot-local folder so that we can check the changes outside VSCode even before they are merged to the master branch.

So that we can still get the vsix from the libs-snapshot-local folder, even if all the tests don't pass, for testing purposes.